src/components/MainSection/MainSection.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { MdLocationOn } from 'react-icons/md';
+
+import { useWeatherInfo } from 'WeatherInfoProvider';
+import WeatherIcon from 'components/WeatherIcon/WeatherIcon';
+import * as utilsFunc from 'utils/convertFunc';
+
+import styles from './MainSection.module.css';
+
+function MainSection(props) {
+  const {location, current} = useWeatherInfo();
+  const dateString = current && utilsFunc.formatDateString(current.date);
+  const temperature = current?.temperature && Math.round(utilsFunc.convertKevinToCelcius(current.temperature));
```

Comment:
```suggestion
  const dateString = current 
    ? utilsFunc.formatDateString(current.date)
    : "";
  const temperature = current.temperature
    ? Math.round(utilsFunc.convertKevinToCelcius(current.temperature))
    : "";
```

to clearly know the value of the variable when it is falsy